docs/source/markdown/podman-create.1.md

```diff
@@ -410,6 +410,11 @@ Container host name
 
 Sets the container host name that is available inside the container. Can only be used with a private UTS namespace `--uts=private` (default). If `--pod` is specified and the pod shares the UTS namespace (default) the pod's hostname will be used.
 
+#### **--hostuser**=*name*
+
+Add host user account to /etc/passwd within container. Username or UID must
+exists on the host system.
```

Comment:
Please make it more clear that this does not set the user in the container to be the new UID or GID.

Does this work with rootless Podman, considering UID/GID mapping means the UID/GID won't actually match the one on the host? Probably need a warning about that.

Also should mention explicitly that UID is allowed.

Reply:
Should we change the UID to match the UID on the host? within the user namespace?
IE Adding dwalsh (UID=3267) would end up as UID=0 within the container.